docs/docs/themes/getting-started.md

```diff
@@ -1,35 +1,56 @@
 ---
-title: Getting Started
+title: Using a Gatsby theme
```

Comment:
```suggestion
title: Using a Gatsby Theme
```

Should we change the filename to match the new title (typically what we recommend for SEO)? We might also want a redirect (could check analytics)

Reply:
Yeah, should change the filename. No reason not to add a redirect, just in case it's linked somewhere.